verification/store.go

```diff
@@ -64,6 +64,10 @@ func LoadX509TrustStore(path string) (*X509TrustStore, error) {
 		trustStore.Certificates = append(trustStore.Certificates, certs...)
 	}
 
+	if len(trustStore.Certificates) < 1 {
+		return nil, fmt.Errorf("trust store %q has no x509 certificates", path)
+	}
+
```

Comment:
I'm debating if this function should throw an error or the caller should account for this condition. The caller has more context such as value of `trustpolicy.truststore` value, and in the case where we extend policy to support specifying multiple trust stores, an empty store is a valid condition.

Reply:
> in the case where we extend policy to support specifying multiple trust stores, an empty store is a valid condition.

Why is it a valid case?

Reply:
Discussed offline. Keeping it as is for now. Will remove the validation when needed.